### ISSUE TITLE
Add possibility to exclude single files

### ISSUE THREAD

**Issue by Stefan-Ilic** [rubicon-oss/LicenseHeaderManager#115]
As mentioned in #114, it would be useful to have the possibility to exclude single files from license header generation. This could be achieved by making it possible to add file names to `.licenseheader` files, in addition to file extensions.

One possible issue arising could be the handling of multiple files with the same name. This can be circumvented by placing a `.licenseheader` file into the specified folder where the file to exclude lies. However, this would inadvertently result in the duplication of license information from the main file to cover other files within that folder.

A better solution would perhaps be to provide a specific path to the file to ignore in the main `.licenseheader` file.

**Comment by VPKSoft** [comment]
Hi,
Thank you for this, I'll vote for the path thing then 👍

**Comment by VPKSoft** [comment]
I did a quick view over the code but with that I'm not making a PR.
To have a single file to have or not to have a license header I thought to replace a comparison in the file LicenseHeaderReplacer.cs at row 208 from this:
```cs
      var language = _licenseHeaderExtension.LanguagesPage.Languages
          .Where (x => x.Extensions.Any (y => item.Name.EndsWith (y, StringComparison.OrdinalIgnoreCase)))
          .FirstOrDefault();
```

to this:
```cs
      var language = _licenseHeaderExtension.LanguagesPage.Languages
          .FirstOrDefault(x => x.Extensions.Any (y => item.Name.EndsWith (y, StringComparison.OrdinalIgnoreCase) 
            || y.Equals(item.Name.TrimStart('.'), StringComparison.OrdinalIgnoreCase)));
```

**Comment by MichaelKetting** [comment]
@VPKSoft Yes, this would work easily for excluding files with the same name across the entire scope of the respective licenseheader-file. The problem is that you would apply this match for all files that have the same name and this might not be what everyone wants and therefor we need the complete solution that also takes into account the entire file path, not just the file's name.
